webplugin/js/app/zendesk-chat-service.js

```diff
@@ -242,6 +242,13 @@ var onTabClickedHandlerForZendeskConversations = function (event) {
     console.log("onTabClicked from zendesk: ", event, MCK_GROUP_MAP[event.tabId]);
     if (kommunicate._globals.zendeskChatSdkKey) {
         var currentGroupData = MCK_GROUP_MAP[event.tabId];
+        var conversationInfo = {
+          groupId: event.tabId,
+          metadata: {
+            "source" : "zopim"
+          }
+        }
+        Kommunicate.updateConversationMetadata(conversationInfo);
```

Comment:
use some formatter extension in VS code, and before commit do a formatting.
global spacing of file is 4 spaces.